### ISSUE TITLE
LODES8

### ISSUE THREAD

**Issue by bnglasner** [jamgreen/lehdr#30]
I am curious if the package will be adapted to the new LODES 8.0 data which includes 2020 data and recasts all historical LODES years into 2020 census blocks.

**Comment by dillonma** [comment]
Great question. I just realized that we need to add in a parameter to grab different LODES versions. Right now, it's hard-coded (doh) to grab LODES 7 data. We'll work on that. 

I looked at the LODES 8 technical docs and the folder structure and file structure are the same as LODES 7, so it should be an easy update.

**Comment by chuckpurvis** [comment]
Great! I came across the Census Bureau's April 4, 2023 posting on the release of the 2020 data! I just tried lehdr with 2020, and it bombs, due to LODES8 vs LODES7.

Yes, a parameter to choose between LODES7 (2010 Census-based geographies) and LODES8 (2020 Census-based geographies) would be an important addition! Somehow caveat that 2020 data isn't available for older (2010) geographies? (LODES7). Thanks. Chuck
